README.md

```diff
@@ -340,7 +340,8 @@ e.g. python -m BTVNanoCommissioning.utils.compile_jec Winter22Run3 jec_compiled
 - data/MC comparisons
 :exclamation_mark: If using wildcard for input, do not forget the quoatation marks! (see 2nd example below)
 
-You can specify `-v all` to plot all the variables in the `coffea` file
+You can specify `-v all` to plot all the variables in the `coffea` file, or use wildcard options (e.g. `-v *DeepJet*` for the input variables containing `DeepJet`)
```

Comment:
For wildcard option, quotation marks have to be used such that the argument will be recognized as string and not interpreted in the wrong way by the bash shell, i.e. change it to `-v "*DeepJet*"`